src/core/modules/NomenclatureItem.cpp

```diff
@@ -250,7 +252,9 @@ Vec3d NomenclatureItem::getJ2000EquatorialPos(const StelCore* core) const
 	jde = core->getJDE();
 	const Vec3d equPos = planet->getJ2000EquatorialPos(core);
 	// Calculate the radius of the planet. It is necessary to re-scale it
-	const double r = planet->getEquatorialRadius() * static_cast<double>(planet->getSphereScale());
+	double r = planet->getEquatorialRadius() * static_cast<double>(planet->getSphereScale());
+	if (nType==NomenclatureItem::niPole)
+		r = planet->getPolarRadius() * static_cast<double>(planet->getSphereScale());
```

Comment:
This should not really be the final solution. The complete solution valid for ellipsoid bodies is in the Explanatory Supplement (3rd) pp 131ff (solve for r), or Astr.Algorithms (2nd) p.82 (solve for rho). It may be useful to add a  method to the Planet class. 
Hmm, I have done that already...
Yes, StelObserver::getTopographicOffsetFromCenter(void) has the code. We could move the computational part of this to the Planet class, and change to  a calling wrappers in StelObserver. If needed, I can do that later tonight.

Reply:
Please do it.